packages/ckeditor5-ui/src/panel/balloon/balloonpanelview.ts

```diff
@@ -1145,6 +1174,20 @@ export function generatePositions( options: {
 					...config
 				}
 			};
+		},
+
+		// ------- Hidden
+
+		viewportHidden: () => {
```

Comment:
The names of positions were written in such a way that they describe where the balloon will show up. For instance `eastArrowWest` means the balloon will show up east of its target with the arrow poining west. `viewportStickyNorth` means that the balloon will be sticky to the north of the viewport. `viewportHidden` then means...? I think plain `hidden` would make more sense here.